pkgs/by-name/js/json-search/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib
+  , stdenv
+  , fetchFromGitHub
+  , json_c
+  , pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "json-search";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "cosmo-ray";
+    repo = "json-search";
+    rev = version;
+    hash = "sha256-bzo1qHYY9mVUFG0s8D68Y/t4UoBDBcKEF1+tCR2zTvY=";
+  };
+
+  nativeBuildInputs = [
+    json_c
```

Comment:
Shouldn't this be on buildInputs?

Reply:
Indeed and I forgot glibc too. Good catch!